.github/workflows/release.yaml

```diff
@@ -39,19 +39,31 @@ jobs:
       - name: Get tag and repo
         run: |
           echo "TAG=${GITHUB_REF#refs/tags/}" >> $GITHUB_ENV
-          echo "REPO=$(echo $GITHUB_REPOSITORY | awk '{print tolower($0)}')" >> $GITHUB_ENV
+          echo "REPO_OWNER=$(echo $GITHUB_REPOSITORY_OWNER | awk '{print tolower($0)}')" >> $GITHUB_ENV
 
       - name: Build eraser-manager
         run: make docker-build-manager \
           CACHE_FROM=type=gha,scope=eraser-manager \
           CACHE_TO=type=gha,scope=eraser-manager,mode=max \
-          MANAGER_IMG=${{ env.REGISTRY }}/${REPO}-manager:${TAG}
+          MANAGER_IMG=${{ env.REGISTRY }}/${REPO_OWNER}/eraser-manager:${TAG}
 
       - name: Build eraser
         run: make docker-build-eraser \
           CACHE_FROM=type=gha,scope=eraser-node \
           CACHE_TO=type=gha,scope=eraser-node,mode=max \
-          ERASER_IMG=${{ env.REGISTRY }}/${REPO}:${TAG}
+          ERASER_IMG=${{ env.REGISTRY }}/${REPO_OWNER}/eraser:${TAG}
+
+      - name: Build collector
+        run: make docker-build-collector \
+          CACHE_FROM=type=gha,scope=collector \
+          CACHE_TO=type=gha,scope=collector,mode=max \
+          COLLECTOR_IMG=${{ env.REGISTRY }}/${REPO_OWNER}/collector:${TAG}
+
+      - name: Build Trivy scanner
+        run: make docker-build-trivy-scanner \
+          CACHE_FROM=type=gha,scope=trivy-scanner \
+          CACHE_TO=type=gha,scope=trivy-scanner,mode=max \
+          TRIVY_SCANNER_IMG=${{ env.REGISTRY }}/${REPO_OWNER}/trivy-scanner:${TAG}
```

Comment:
Can we change the default `TRIVY_SCANNER_IMG` in the Makefile to match this? It's currently `eraser-trivy-scanner`

Reply:
good catch, updated name to `eraser-trivy-scanner`